src/features/features.controller.ts

```diff
@@ -35,6 +47,16 @@ export class FeaturesController {
         summary:
             "Adds a new feature for a team given a teamId (int) and that the user is logged in.",
     })
+    @ApiResponse({
+        status: HttpStatus.NOT_FOUND,
+        description: "Feature Category with given ID does not exist.",
+        type: NotFoundErrorResponse,
+    })
+    @ApiResponse({
+        status: HttpStatus.CREATED,
+        description: "Successfully created a new feature.",
+        type: FeatureResponse,
+    })
```

Comment:
Should we add ApiResponse for Unauthorized? I see an undocumented response.
<img width="1357" alt="Screenshot 2024-01-12 at 12 06 35 PM" src="https://github.com/chingu-x/chingu-dashboard-be/assets/80556079/624a72b7-e6c4-4449-84a2-d3969fd90b48">